### PR TITLE
DevFest Warsaw 2018 added

### PR DESCRIPTION
**Pull request by blipinsk**

### CODE REVIEW

File: _conferences/devfest-warsaw-2018.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: "GDG DevFest"
+website: https://warsaw.devfest.pl/
+location: Warsaw, Poland
+
+date_start: 2018-12-01
+date_end:   2018-12-02
+
+cfp_start: 2018-10-23
```
Comment (JakeWharton) [inline _conferences/devfest-warsaw-2018.md:9]: No end date?

Comment (blipinsk) [reply _conferences/devfest-warsaw-2018.md:9]: `¯\_(ツ)_/¯` unknown yet (or not known publicly).
I will update it when/if it's known.

Comment (blipinsk) [reply _conferences/devfest-warsaw-2018.md]: (or not known publicly)

Comment (JakeWharton) [reply _conferences/devfest-warsaw-2018.md:9]: Not sure how the site will behave, but let's see

Comment (blipinsk) [reply _conferences/devfest-warsaw-2018.md:9]: C4P badge isn't visible. I will temporarily set it to the starting date of the conference.

**Pull request merged**